pkg/test/resourcefixture/testdata/basic/sql/v1beta1/sqlinstance/sqlserverinstancebasic/_generated_object_sqlserverinstancebasic.golden.yaml

```diff
@@ -3,24 +3,24 @@ kind: SQLInstance
 metadata:
   annotations:
     cnrm.cloud.google.com/management-conflict-prevention-policy: none
-    cnrm.cloud.google.com/mutable-but-unreadable-fields: '{"spec":{"rootPassword":{"value":"4321"}}}'
```

Comment:
These (line 6-7) are actually the annotations used to support mutable-but-unreadable.

Reply:
Thanks for the info on this. Will implement support for this feature in follow-on PR.